bazel/cc_toolchains/clang_cc_toolchain_config.bzl

```
@@ -642,6 +642,19 @@ def _impl(ctx):
         ],
     )
 
+    # A feature that enables poisoning of value stores to detect use after
+    # potential reallocation bugs.
```

Maybe include a TODO, mostly to document that we don't expect this to stay forever?

```suggestion
    # A feature that enables poisoning of value stores to detect use after
    # potential reallocation bugs.
    #
    # TODO: Remove this and leave poisoning always on once these bugs are
    # fixed.
```